rust-runtime/aws-smithy-checksums/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "aws-smithy-checksums"
-version = "0.60.12"
+version = "0.62.0"
```

Comment:
Are we skipping `0.61.0`?

Reply:
`0.61.0` was already published and yanked so to avoid any confusion I made this `0..62.0` instead of `0.61.1`, but I don't have a strong opinion here and am happy to change it.